php/public/options-form-submit.js

```diff
@@ -22,7 +24,9 @@ document.addEventListener("DOMContentLoaded", function () {
         });
 
         // Show or hide submit button based on changes
-        optionsFormSubmit.style.display = hasChanges ? 'block' : 'none';
+        optionsFormSubmit.forEach(optionsFormSubmit => {
+            optionsFormSubmit.style.display = hasChanges ? 'block' : 'none';
```

Comment:
I think this would work incorrectly as it would enable both buttons whenever one of the checkboxes in either form is clicked. How can we fix this?

Reply:
found a solution